tests/unit/ReferenceListTest.php

```diff
@@ -147,14 +133,6 @@ public function testRemoveReference( ReferenceList $array ) {
 			$this->assertFalse( $array->hasReference( $element ) );
 			$this->assertEquals( --$elementCount, count( $array ) );
 		}
-
-		$elements = $this->getElementInstances();
-		$element = array_shift( $elements );
-
-		$array->removeReference( $element );
-		$array->removeReference( $element );
-
-		$this->assertTrue( true );
```

Comment:
I tried again to be sure. No bad things happen in strict mode. CI does not complain, too.